jest.config.js

```diff
@@ -0,0 +1,184 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
Do we want all this commented code?

Reply:
Doesn't harm, it's was created by the init script and makes easier to find the options. However, I don't mind to leave it or remove it.

what do you think?

Reply:
Leave it in, useful if we need to change later